buildspec.yaml

```diff
@@ -18,9 +18,7 @@ phases:
       - /bin/bash ./fetch-dotnet-snk.sh
   build:
     commands:
-      # we bump here so that our master version won't be identical to the latest published version during tests.
-      # otherwise this causes problems with verdaccio mirroring.
-      - '[ ${GIT_BRANCH} = ${REGRESSION_TESTS_BRANCH} ] && /bin/bash ./bump-candidate.sh'
+      - /bin/bash ./scripts/bump-candidate.sh
```

Comment:
I'd rather keep the flag visible in the buildspec.

The actual problem here is the `&&`. We should replace this with `if ${CANDIDATE_BUMP}; then ./bump-candidate.sh; fi`

Reply:
Agree.